r128.h

```diff
@@ -677,10 +677,10 @@ static R128_U64 r128__umul64(R128_U32 a, R128_U32 b)
 // 64/32->32
 static R128_U32 r128__udiv64(R128_U32 nlo, R128_U32 nhi, R128_U32 d, R128_U32 *rem)
 {
-#  if defined(_M_IX86) && (_MSC_VER >= 1920) && !defined(R128_STDC_ONLY)
+#  if defined(_M_IX86) && (_MSC_VER >= 1920) && !defined(R128_STDC_ONLY) && !defined(__MINGW32__)
```

Comment:
This one is actually unnecessary, because it can't simultaneously be `(_MSC_VER >= 1920)` and `(__MINGW32__)`. Otherwise I agree this fixes compiling 32-bit binaries with MinGW.

Reply:
Yep, I have check it and it's excessive indeed, removed.